src/app/examples/examples.routes.ts

```diff
@@ -129,6 +138,44 @@ export const routes: Routes = [
   {
     path: 'list',
     component: ListExampleComponent,
+    children: [
+      {
+        path: 'default',
```

Comment:
Consider using empty for default path:
```suggestion
        path: '',
```